public/js/projectsConstructor.js

```diff
@@ -39,11 +39,11 @@ Project.fetchAll = function() {
     Project.loadAll(JSON.parse(localStorage.projectsList));
     Project.initIndexPage();
   } else {
-    $.get('/data/projects.json', function(response) {
+    $.get('/data/projects.json', ((response) => {
```

Comment:
You have an extra set of parentheses here. The callback does not need to wrapped in parentheses, you can just do
```
$.get('/data/projects.json', response => {
...
});
```